x/core/types/data_request_indexing.go

```diff
@@ -185,9 +185,9 @@ func (s DataRequestIndexing) GetDataRequestsByStatus(ctx sdk.Context, status Dat
 		return nil, nil, 0, err
 	}
 
-	rng := collections.NewPrefixedPairRange[DataRequestStatus, DataRequestIndex](status)
+	rng := collections.NewPrefixedPairRange[DataRequestStatus, DataRequestIndex](status).Descending()
 	if lastSeenIndex != nil {
-		rng.StartExclusive(lastSeenIndex)
+		rng.EndExclusive(lastSeenIndex)
```

Comment:
Is this correct? Isn't `lastSeenIndex` supposed to specify the starting point of the range?

Reply:
we don't want to include the last seen dr. StartExclusive includes the last seen one.

Reply:
I see, because we traverse in descending order, huh